From 3e8abee449d7c98fb22dd947548378b7649b31f6 Mon Sep 17 00:00:00 2001
From: Mingcong Bai <jeffbai@aosc.io>
Date: Thu, 4 Apr 2024 00:16:06 +0800
Subject: [PATCH] HACK(cmake): drop broken Wayland modules

Wayland modules are out-of-sync with upstream KF5 modules, drop them.

Ref: https://gitlab.archlinux.org/archlinux/packaging/packages/deepin-qt5platform-plugins/-/blob/a6d06a904411502d3c685eba479cc845b6e044e5/PKGBUILD#L22
---
 CMakeLists.txt | 3 ---
 1 file changed, 3 deletions(-)

diff --git a/CMakeLists.txt b/CMakeLists.txt
index 08fec8d..74dfdfd 100644
--- a/CMakeLists.txt
+++ b/CMakeLists.txt
@@ -51,9 +51,6 @@ endif ()
 set(LIBRARY_OUTPUT_PATH ${CMAKE_BINARY_DIR}/plugins/platforms)
 
 add_subdirectory(xcb)
-if("${PROJECT_VERSION_MAJOR}" STREQUAL "5")
-    add_subdirectory(wayland)
-endif()
 if(BUILD_TESTING AND NOT ${CMAKE_SYSTEM_PROCESSOR} MATCHES "loongarch")
     enable_testing()
     add_subdirectory(tests)
-- 
2.44.0

